src/Data/Functor/Foldable.hs

```diff
@@ -29,10 +29,6 @@ module Data.Functor.Foldable
   -- * Base functors for fixed points
     Base
   , ListF(..)
-  -- * Fixed points
-  , Fix(..), unfix
-  , Mu(..), hoistMu
-  , Nu(..), hoistNu
```

Comment:
If we're nor re-exporting those, don't we need something bigger than a minor version bump?

Reply:
Ah, my mistake, I saw the bump from 0.10 to 0.10.1 in `.travis.yml` and I assumed we were doing a minor version bump, but I now see that the version number in `recursion-schemes.cabal` hasn't been bumped, so we'll do whatever version bump is necessary.